package.json

```diff
@@ -93,13 +93,13 @@
         "icon": "$(add)"
       },
       {
-        "command": "onevscode.flip-cfg-to-gui",
+        "command": "one.editor.open",
```

Comment:
If we introduce new type of custom editor, we cannot distinguish from it.
So I think adding `cfg` word seems better :)

```suggestion
        "command": "one.editor.openCfg",
```

Reply:
Do we plan to? I thought we plan to add more backend but not a cfg editor..

Reply:
What about renaming it later then? :-D